Formula/erlang@22.rb

```diff
@@ -63,8 +64,12 @@ def install
     system "./configure", *args
     system "make"
     system "make", "install"
-
-    (lib/"erlang").install resource("man").files("man")
+    # Homebrew flattens the directory tree of the archive in case there are no more files in the root, so we loose man and need to move the individual subdirs
+    (lib/"erlang/man").install resource("man").files("man1")
+    (lib/"erlang/man").install resource("man").files("man3")
+    (lib/"erlang/man").install resource("man").files("man4")
+    (lib/"erlang/man").install resource("man").files("man6")
+    (lib/"erlang/man").install resource("man").files("man7")
```

Comment:
Is is not enough to just do `(lib/"erlang").install resource("man")`?

Reply:
No, because `man` gets removed by Homebrew when it is smart while unpcaking the archive.